workers/unity/Packages/com.improbable.gdk.core/Authority/AuthorityComponents.cs

```diff
@@ -20,12 +20,14 @@ public struct NotAuthoritative<T> : IComponentData where T : ISpatialComponentDa
 
     /// <summary>
     ///     ECS component denotes that this worker will lose authority over <see cref="T"/> imminently.
+    ///     If AcknowledgeAuthorityLoss is set then authority handover will complete before the handover timeout.
```

Comment:
when exactly will it be completed? as soon as the ack is received?

Reply:
Depends on the runtime. Can't give an accurate time here. Just that the runtime will do it as fast as it can.

Reply:
Distributed systems basically.

Reply:
fair enough